### PR TITLE
Added coverage to contributing guide, and badges to README

### PR DESCRIPTION
**Pull request by slintes**

### CODE REVIEW

**Comment by cynepco3hahue** [comment]
Looks great!

/lgtm

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -37,14 +37,19 @@ contribution. See the [DCO](DCO) file for details.
...
+# Unit Testing and Coverage
 
 Unit testing allows us to validate our reconcile loops under specific conditions to ensure no unexpected changes in behavior are introduced.
 
 * Use mock clients to simulate reconcile loops.
 * Execute reconcile loop to validate execution under specific conditions.
 * Validate reconcile execution by introspecting API calls that were made during the execution. [Example](https://github.com/operator-framework/operator-sdk-samples/blob/master/go/memcached-operator/pkg/controller/memcached/memcached_controller_test.go)
 
+Unit tests can be executed with `make unittests`. This will also create unit test coverage reports, their location will be reported.
+Every PR will also run unit tests with coverage, and the status will be reported on the PR. Make sure the coverage doesn't descrease,
```
Comment (MarSik) [inline CONTRIBUTING.md:49]:
```suggestion
Every PR will also run unit tests with coverage, and the status will be reported on the PR. Make sure the coverage doesn't decrease,
```

Comment (slintes) [reply CONTRIBUTING.md]: thx, fixed

**Pull request merged**